.github/workflows/all-post-commit-workflows.yaml

```diff
@@ -17,9 +17,11 @@ jobs:
   static-checks:
     uses: ./.github/workflows/all-static-checks.yaml
     secrets: inherit
-  build-wheels:
+  build-and-test-wheels:
```

Comment:
Do you think we could skip the wheel tests here since we are already going to test them with unit-tests? 
That will cut down time before the tests can be kicked off.

Reply:
We could potentially cut out the silicon tests, but I'd like to keep the host tests.
They're very fast tests, anyway. <5 mins?

Reply:
If we could move them along the other tests so that they do not block the queueing of the other all commit tests, that would be awesome.

Reply:
made an issue here for later trakcing: https://github.com/tenstorrent/tt-metal/issues/12538